src/redux/reducers/group-reducer.js

```diff
@@ -24,6 +25,7 @@ const setLoadingState = state => ({ ...state, isLoading: true });
 const setRecordLoadingState = state => ({ ...state, isRecordLoading: true, selectedGroup: { ...state.selectedGroup, loaded: false }});
 const setRecordRolesLoadingState = state => ({ ...state, isRecordRolesLoading: true, selectedGroup: { ...state.selectedGroup, loaded: false }});
 const setGroups = (state, { payload }) => ({ ...state, groups: payload, isLoading: false });
+const setSystemGroup = (state, { payload }) => ({ ...state, systemGroup: payload.data.filter(group => group.platform_default)[0] });
```

Comment:
Conditional accessor might be good here so we don't break the UI
```suggestion
const setSystemGroup = (state, { payload }) => ({ ...state, systemGroup: payload?.data?.filter(group => group?.platform_default)?.[0] });
```